rotkehlchen/api/rest.py

```diff
@@ -3946,14 +3946,15 @@ def delete_manual_latest_price(
                 asset=asset,
             )
         try:
-            to_asset = GlobalDBHandler().delete_manual_latest_price(asset=asset)
+            pairs_to_invalidate = GlobalDBHandler().delete_manual_latest_price(asset=asset)
         except InputError as e:
             return api_response(
                 result=wrap_in_fail_result(message=str(e)),
                 status_code=HTTPStatus.CONFLICT,
             )
 
-        Inquirer().remove_cached_current_price_entry(cache_key=(asset, to_asset))
+        for pair in pairs_to_invalidate:
```

Comment:
I was gonna say to not use a for loop but pass the list of pairs here to invalidate so you avoid for loop operations on the DB But after checking this is not a DB operation so it's fine